WorkspaceServer/WorkspaceServer.csproj

```diff
@@ -38,20 +38,20 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Buildalyzer" Version="3.0.0" />
-    <PackageReference Include="Buildalyzer.Workspaces" Version="3.0.0" />
+    <PackageReference Include="Buildalyzer" Version="3.2.2" />
+    <PackageReference Include="Buildalyzer.Workspaces" Version="3.2.2" />
     <PackageReference Include="Clockwise" Version="1.0.261-beta" />
-    <PackageReference Include="FSharp.Core" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.6.0" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.6.0" />
-    <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.Workspaces" Version="3.6.0" />
-    <PackageReference Include="Microsoft.CodeAnalysis.Features" Version="3.6.0" />
-    <PackageReference Include="Microsoft.CodeAnalysis.Scripting" Version="3.6.0" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Features" Version="3.6.0" />
-    <PackageReference Include="Microsoft.CodeAnalysis.Scripting.Common" Version="3.6.0" />
-    <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="3.6.0" />
+    <PackageReference Include="FSharp.Core" Version="5.0.2" />
```

Comment:
Don't specify F# package versions; they're all handled in the root `Directory.Build.targets`.